packages/boolean/StoryComponent.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react'
+import useBoolean from '.'
+
+export default () => {
+  const [value, toggle] = useBoolean(true)
+
+  return (
+    <React.Fragment>
+      <p>The value is now {value ? 'true' : 'false'}</p>
+      <button onClick={() => toggle()}>Toggle</button>
+    </React.Fragment>
+  )
+}
```

Comment:
This is just a test, if you have any ideas, we can do differently!

Reply:
Nice that's cool, we can add some shared 'demo' styles later.